### ISSUE TITLE
Probably incorrect usage of tabindex={-1} for List

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#1549]
Probably one can remove tabIndex from [here](https://github.com/gravity-ui/uikit/blob/8de16536b7541835d58cc651a3ab4248f0e1ebf0/src/components/List/List.tsx#L152), as div element is not focusable already.

**Comment by amje** [comment]
Removing it could potentially break existing code
